File: Park20_App/src/app/_dtos/Customer.dto.ts
import { VehicleDTO } from "./Vehicle.dto";

export interface CustomerDTO {
  id? : string;
  username : string;
  password: string;
  email: string;
  name : string
  nif: string;
  handicapped: boolean;
  vehicles: VehicleDTO[];
  payments: any;
  language?: string;
  wallet?: {amount: string};
}

File: Customer_Microservice/src/main/java/park20/Customer_Microservice/dto/CustomerDTO.java
package park20.Customer_Microservice.dto;

import park20.Customer_Microservice.domain.ParkyWallet.ParkyWallet;

import java.io.Serializable;

public class CustomerDTO implements Serializable {

    public String id;
    public String email;
    public String username;
    public String password;
    public String name;
    public String nif;
    public boolean handicapped;
    public CustomerPaymentMethodDTO[] payments;
    public VehicleDTO[] vehicles;
    public String language;

    public ParkyWalletDTO wallet;
}
